web-www/styles/font.css

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css?family=Muli:800,900&display=swap');
```

Comment:
so now we need those web-components css files twice so we can import them directly from _app?

Reply:
They're not imported twice, they used to be imported in the [RegenThemeProvider](https://github.com/regen-network/regen-web/blob/main/web-components/src/theme/RegenThemeProvider.tsx#L34) but we're not using this provider here as it was not SSR compatible.

Reply:
Yes sure, we'll no longer need the old provider once `web-registry` will be migrated to nextjs.